dotnet/src/Microsoft.Agents.AI.Purview/Models/Common/AiAgentInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Agents.AI.Purview.Models.Common;
+
+/// <summary>
+/// Info about an AI agent associated with the content.
+/// </summary>
+internal sealed class AiAgentInfo
```

Comment:
```suggestion
internal sealed class AIAgentInfo
```
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

If you change the class name, don't forget to change file name also.